assets/queries/cloudFormation/security_groups_allows_unrestricted_outbound_traffic/query.rego

```diff
@@ -0,0 +1,17 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   resources := document[i].Resources[name]
+   properties := resources.Properties
```

Comment:
Just for sanity sake, check the resource if its a SecurityGroup, in case some resource attributes change